src/charts/stacked-area.js

```diff
@@ -1019,11 +1019,11 @@ define(function(require){
             cleanDataPointHighlights();
 
             // ensure order stays constant
-            values = values
```

Comment:
You still need the values here, right?

Reply:
doesn't seem like it. It works without it.

Reply:
OK then